app/src/components/adminConsole/manageTeam/SnappablesTable.tsx

```diff
@@ -1,35 +1,110 @@
+            },
+            {
+                Header: "Username",
+                accessor: "username",
+            },
+            {
+                Header: "Snaps Received",
+                accessor: "numSnaps",
+            },
+        ],
+        []
+    );
+
+    const data = React.useMemo(
+        () =>
+            props.snappables
+                .sort((a: Snappable, b: Snappable) =>
+                    a.fullName.localeCompare(b.fullName)
+                )
+                .map((p: Snappable) => ({
+                    id: p.id,
+                    email: p.email,
+                    fullName: p.fullName,
+                    username: p.username,
+                    numSnaps: countSnapsForUser(p.id, props.currentSnaps),
+                })),
+        [props]
+    );
+
+    const table = useTable(
+        { defaultColumn, columns, data },
+        useSortBy,
+        useFlexLayout
+    );
 
     return (
-        <table className="table  table-hover">
+        <table {...table.getTableProps()} className="table  table-hover">
             <thead>
-                <tr>
-                    <th scope="col">Email</th>
-                    <th scope="col">Full Name</th>
-                    <th scope="col">Username</th>
-                    <th scope="col">Snaps Received</th>
-                </tr>
+                {table.headerGroups.map((headerGroup) => (
+                    <tr {...headerGroup.getHeaderGroupProps()}>
+                        {headerGroup.headers.map((column) => (
+                            <th
+                                {...column.getHeaderProps(
+                                    column.getSortByToggleProps()
+                                )}
+                                scope="col"
+                            >
+                                {column.render("Header")}
+                                <span>
+                                    {column.isSorted
+                                        ? column.isSortedDesc
+                                            ? " ▾"
+                                            : " ▴"
+                                        : ""}
+                                </span>
+                            </th>
+                        ))}
+                    </tr>
+                ))}
             </thead>
-            <tbody>{rows}</tbody>
+            <tbody {...table.getTableBodyProps()}>
+                {table.rows.map((row) => {
+                    table.prepareRow(row);
+                    return (
+                        <tr {...row.getRowProps()}>
+                            {row.cells.map((cell) => {
+                                return (
+                                    <td {...cell.getCellProps()}>
+                                        <div
+                                            style={{
+                                                overflowWrap: "break-word",
+                                            }}
```

Comment:
Can this go into a css class please, I'd like to avoid inline styles as much as possible.

Reply:
Sure thing, I'll spin it into utilities.scss

Reply:
Have also applied it to the parent `td` now as there's no point in this div